### DISCUSSION TITLE
Multipart seasons

### DISCUSSION THREAD

**Discussion by PassiveLemon** [jellyfin/jellyfin#9814] [Q&A] [answered]
I am currently trying to set up Attack on Titan but I've encountered a problem. AoT has some multipart seasons, (Season 3, Season 3 Part 2). How would I correctly set up these?
Right now I have my file structure as `series/season X/EXX.xxx`.
What I have tried:
 - Putting the other part of the season as a new season (Season 3, Season 4) but that doesn't work correctly
 - Just adding part (Season 3, Season 3 Part 2), but that also doesn't work, It will see them as the same season and combine them into one with duplicate episodes.
 - Lastly, using the file structure `series/season/part X/EXX.xxx` but that has the same result as before.
 
 The only thing I could find online was [this](https://www.reddit.com/r/jellyfin/comments/u1jrhq/how_to_handle_multipart_seasons/) but it was not very helpful

**Comment by Shadowghost** [comment, marked as answer]
You should align your naming with whatever your metadata provide of choice (TmDB by default) uses. Most likely the second part of a season just extends the first part with ongoing episode numbers

**Reply by PassiveLemon** [reply]
Ah okay I see. I just continued the number of the episodes from the previous parts and it seems to find them correctly now. Thanks
